CMakeLists.txt

```diff
@@ -122,3 +122,15 @@ if (TESTING)
   enable_testing()
   add_subdirectory(test)
 endif ()
+
+# update git submodules
+find_package(Git QUIET)
```

Comment:
Why do we need to update submodules on every cmake run? 
I think it is enough to clone some certain commit of remote repo and update submodule as separate PR to cpp-filecoin when required.
Or I didn't get what this cmake is doing

Reply:
It's just for convenience - we should call "git submodule update" to download submodules content after Filecoin repo clone, maybe, it's unnecessarily?

Reply:
I don't think you need it.

Reply:
If it works without this step, pls remove it. We should be forced to update submodules like that each time cmake reruns. What if internet connection is unstable?